qiskit_nature/algorithms/pes_samplers/bopes_sampler.py

```diff
@@ -219,9 +265,78 @@ def _run_single_point(self, point: float) -> EigenstateResult:
         result = self._gss.solve(self._problem)
 
         # Save optimal point to bootstrap
-        if isinstance(self._gss.solver, VariationalAlgorithm):  # type: ignore
-            # at every point evaluation, the optimal params are updated
+        if isinstance(self._gss, GroundStateSolver) and isinstance(
+            self._gss.solver, VariationalAlgorithm
+        ):  # type: ignore
             optimal_params = result.raw_result.optimal_point
             self._points_optparams[point] = optimal_params
 
         return result
+
+    def _run_single_point_excited_solver(self, point: float) -> EigenstateResult:
```

Comment:
Why does this method need to be duplicated for the excited state case?

Reply:
I rolled back on this addition. Not needed anymore